### PR TITLE
Gltf nits

### PR DESCRIPTION
**Pull request by chrisfromwork**
Overview
---
Two changes:
1) Fix gltf test unity scene to have correct file uri
2) Switch to a different png loading mechanism for gltf textures. Old mechanism threw an access denied based on opening a stream.

Changes
---
- Fixes: # .

### CODE REVIEW

File: Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs [hunk 1]
```diff
@@ -130,20 +130,7 @@ private static async Task ConstructTextureAsync(this GltfObject gltfObject, Gltf
...
-                            if (gltfObject.LoadAsynchronously)
-                            {
-                                await stream.ReadAsync(imageData, 0, (int)stream.Length);
-                            }
-                            else
-                            {
-                                stream.Read(imageData, 0, (int)stream.Length);
-                            }
-                        }
+                        imageData = File.ReadAllBytes(path);
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs]: Why remove the async read?

Comment (chrisfromwork) [reply Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs]: It doesn't work when built in visual studio. You get an access denied when attempting to open the stream. We could duplicate this to a temp file but it seemed higher priority to get this code in with follow up load async improvements in a different issue.

Comment (chrisfromwork) [reply Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs]: Specifically an UnauthorizedAccessException

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs]: Please revert this. I've got a proper fix

**Pull request merged**